### ISSUE TITLE
Error login to confirmation. help me pls 

### ISSUE THREAD

**Issue by hungsnt43** [winauth/winauth#439]
![11](https://cloud.githubusercontent.com/assets/22117156/18410754/ec1a894a-7793-11e6-8b2f-6c7d76c513da.png)

**Comment by winauth** [comment]
Right click and do a sync, then try again.

**Comment by lnwc** [comment]
I think something happened/is still happening on Steam's end since about 24 hours ago. Every time I go right click->confirmations I have the login dialog. Sometimes it logs in, sometimes it does nothing, sometimes it spits the same error hungsnt43 posted. Even after a sync. When I get through and close the confirmations dialog, I have to login all over again the next time I try to go there. I thing related is the thing where https://steamstat.us/ keeps alternating between Steam Community: Normal and Steam Community: 100% Failed Trades. We'll probably have to wait this out. :p

**Comment by hungsnt43** [comment]
i fixed, i export then import, all ok ^^! (english bad)

**Comment by lnwc** [comment]
It seems that Steam introduced a limit on how many request you can make in a given time. Looking around led me https://github.com/Jessecar96/SteamBot/issues/1048  here where someone unearths the message Steam now gives you when you make too many requests.

**Comment by Siddxa** [comment]
yes, its work -  export then import, close and open + sync  = shamanism =)
